src/main/java/Espresso.java

```diff
@@ -0,0 +1,140 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Espresso {
+    private ArrayList<Task> tasks ;
+    private int count;
+
+    public Espresso() {
+        tasks = new ArrayList<>();
+        count = 0;
+    }
+
+    void addToList(String str) {
+        Task task = null;
+        try {
+                if (str.startsWith("todo")) {
```

Comment:
You may change the if-statement indentation to indentation to once (4 spaces) instead of twice.